source/installation-guide/installing-wazuh-agent/deployment_variables/linux/deployment_variables_yum.rst

```diff
@@ -47,41 +47,41 @@ Examples:
 .. code-block:: console
 
      # WAZUH_MANAGER="10.0.0.2" WAZUH_REGISTRATION_PASSWORD="TopSecret" \
-          WAZUH_AGENT_NAME="yum-agent" yum install wazuh-agent
+          WAZUH_AGENT_NAME="yum-agent" yum install wazuh-agentC
```

Comment:
I don't think that we have a `wazuh-agentC`. Please be careful

Reply:
My bad, sorry.